packages/core/helper-plugin/src/components/DynamicTable/v2/Table.js

```diff
@@ -0,0 +1,422 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Flex,
+  Button,
+  Typography,
+  Table as TableCompo,
```

Comment:
```suggestion
  Table as DSTable,
```

Clearer name, "compo" could be anything.